### ISSUE TITLE
metadata_win is empty

### ISSUE THREAD

**Issue by dev-zero** [stepping-stone/sepiola#60]
We have to figure out whether this is actually a regression from 2.3 or not, but anyway...

The following happened for the restore part in the test case

  http://wiki.sepiola.org/index.php/Sepiola_Backup#Windows_2

According to the message, `metadata_win` is invalid, actually, it seems to be empty.

Now, besides fixing the issue at hand, we first have to figure out why there is no error message when creating the `metadata_win`.

```
Establishing connection ...
Validating server's fingerprint ...
Server's fingerprint is correct.
Login with key was successful
Connection established.

Downloading files and/or directories ...

Downloading ./
Downloading C:\
Downloading C:\Users\
Downloading C:\Users\storm-win7\
Downloading C:\Users\storm-win7\Downloads\
Downloading C:\Users\storm-win7\Downloads\sepiola_tests_win\
Downloading C:\Users\storm-win7\Downloads\sepiola_tests_win\special_names\
Downloading C:\Users\storm-win7\Downloads\sepiola_tests_win\special_names\testördnerchen\
Downloading C:\Users\storm-win7\Downloads\sepiola_tests_win\special_names\testördnerchen\testfäilchen
9 files and/or directories downloaded
Applying Metadata
Downloading metadata_win

1 files and/or directories downloaded
Preparing metadata

SetACL finished with error(s):
SetACL error message: A read operation from the backup/restore file failed
Restore done.
```

**Comment by philippoehler** [comment]
`Establishing connection ...
Validating server's fingerprint ...
Server's fingerprint is correct.
Login with key was successful
Connection established.
Taking filesystem snapshot, this might take some time ...


Filesystem snapshot successfully taken
Creating/validating server directories

Skipping /philipp-PC/
1 files and/or directories processed

Skipping /philipp-PC/
Skipping /philipp-PC/.sepiola_backup/
Skipping /philipp-PC/backup/
3 files and/or directories processed

Uploading /qt_temp.qHp844
1 files and/or directories processed

Uploading /qt_temp.Uhg844
1 files and/or directories processed
estimating backup size
Uploading files and directories

Skipping /Users/
Skipping /Users/philipp/
Skipping /Users/philipp/Documents/
Skipping /Users/philipp/Documents/syncMe.txt
4 files and/or directories processed
Getting backup content meta data

Downloading backupContent
1 files and/or directories downloaded
Uploading backup content meta data

Uploading /backupContent

1 files and/or directories processed
Uploading backup info data

Uploading /backupTime

1 files and/or directories processed
Getting permission meta data

Downloading metadata_win
1 files and/or directories downloaded

Uploading permission meta data

Uploading /metadata_win
1 files and/or directories processed
Backup succeeded.

Uploading /qt_temp.aem844

1 files and/or directories processed
==================================================`


It seams there is no SetACL error in previous version. But the `metadata_win` folder on the server is still empty after successful backup. 


*    Tested version: `sepiola-2.3.5-5-g0c91379-win32.exe`
*    Supplier: https://jenkins.stepping-stone.ch/download/sepiola/
*    Environment: Windows7, 32Bit

**Comment by dev-zero** [comment]
ok, re-target for v2.5 as discussed

**Comment by dev-zero** [comment]
it seems that we broke the functionality to store the ACLs at some point (probably the introduction of snapshotting)

**Comment by philippoehler** [comment]
It seems that the ACLs where read from the original folders and files, instead of from the shadow copy.

**Comment by dev-zero** [comment]
confirmed to be fixed in 2.5.0.rc0
